test-data/snake_oil/snake_oil.ert

```diff
@@ -4,8 +4,8 @@ NUM_REALIZATIONS 25
 
 QUEUE_OPTION LOCAL MAX_RUNNING 25
 
-ANALYSIS_SET_VAR IES_ENKF IES_INVERSION 1
-ANALYSIS_SET_VAR STD_ENKF IES_INVERSION 1
+ANALYSIS_SET_VAR IES_ENKF INVERSION 1
```

Comment:
Dont think this is valid, use:
`ANALYSIS_SET_VAR IES_ENKF INVERSION SUBSPACE_EXACT`
`ANALYSIS_SET_VAR STD_ENKF INVERSION SUBSPACE`